model-optimizer/extensions/ops/gather.py

```diff
@@ -98,6 +98,9 @@ def infer(node: Node):
         else:
             node.out_port(0).data.set_shape(out_shape)
 
+        if max(len(data_shape), len(indices_shape)) < 4 and len(out_shape) >= 4:
+            node['reinterp_shape'] = True
```

Comment:
Can you please add a comment why we need this attribute.

Reply:
Done.